api/v1beta1/verticascrutinize_webhook.go

```diff
@@ -149,15 +150,27 @@ func (vscr *VerticaScrutinize) ValidateLogAgeTimes(allErrs field.ErrorList) fiel
 
 	if logAgeOldestTime.After(time.Now()) {
 		err := field.Invalid(prefix.Key(vmeta.ScrutinizeLogAgeOldestTime),
-			scrutinizeLogAgeOldesTime,
+			scrutinizeLogAgeOldestTime,
 			"log-age-oldest-time cannot be set after current time")
 		allErrs = append(allErrs, err)
 	}
 
 	if logAgeOldestTime.After(logAgeNewestTime) {
+		// if log-age-oldest-time is not set, default as 24 hours ago
+		logAgeOldestTimeStr := "log-age-oldest-time is: \"" + logAgeOldestTime.Format("2006-01-02 15 MST") + "\""
+		if scrutinizeLogAgeOldestTime == "" {
+			logAgeOldestTimeStr = "log-age-oldest-time default as 24 hours ago: \"" + logAgeOldestTime.Format("2006-01-02 15 MST") + "\""
```

Comment:
I wonder if we need all these details here. They can be added in the documentation

Reply:
It would be helpful to print the default values if not set, or user would be confused why it's not valid.

Yes, the default values will be added into the doc.